pyvo/dal/sia.py

```diff
@@ -46,7 +46,7 @@
 
 
 def search(
-        url, pos, size=1.0, format='all', intersect="overlaps", verbosity=2,
+        url, pos, size=1.0, format=None, intersect=None, verbosity=None,
```

Comment:
you'll need to update the default for `format` in the docstring below, too. The rest looks good.

Reply:
Oops, good point!  There are 2 other places in the code where the docstring mentions the default, but the function signature default is `None`.  I could edit them all to indicate that the default is a server-side default, but then for consistency I suppose I should add that same info for `intersect` and `verbosity`.  It may help a user to know the server-side default, so maybe it's worth the extra words, but I'm not sure.

Reply:
`format` and `intersect` are fine as those are the service defaults as well so behaviour is unchanged.

I couldn't find a default for `verbosity` in the spec so I'm not sure if this changes the behaviour. But the spec says that "Services that do not support this parameter must permit it to be present without error." so it shouldn't create any problems if present.

Reply:
You're right about `verbosity`.  I was remembering that it defaults to 2 for Simple Cone Search.  To minimize the potential changes in behavior to standard-compliant services for this change, I should restore the client-side default for `verbosity`.

Ideally (if all services were fully compliant) none of this should matter, but there's a non-compliant service I want to use that breaks if you specify `format`.  I don't want to have special code in `pyvo` for non-compliant services, but having `pyvo` specify defaults that should be implemented on the server really is unnecessary at best and possibly confusing.  I think the user should have control over what params get submitted in the query.

Reply:
The client can work around the above problem with `format=None`.  That's somewhat workable, but would need to be done as a special case for the offending service.